GLTFSerialization/GLTFSerialization/Schema/Material.cs

```diff
@@ -92,7 +98,25 @@ public static Material Deserialize(GLTFRoot root, JsonReader reader)
 					case "pbrMetallicRoughness":
 						material.PbrMetallicRoughness = PbrMetallicRoughness.Deserialize(root, reader);
 						break;
-					case "commonConstant":
+                    case "pbrSpecularGlossiness":
+                        material.PbrSpecularGlossiness = PbrSpecularGlossiness.Deserialize(root, reader);
+                        break;
+                    case "extensions":
```

Comment:
extensions are handled by the default case. You should let that process the extensions and hook up an extension handler  via extension factory API.

Reply:
Also why are there two ways of processing KHR_materials_pbrSpecularGlossiness